src/main/java/com/google/api/codegen/transformer/SurfaceTransformerContext.java

```diff
@@ -100,7 +100,10 @@ public MethodTransformerContext asMethodContext(Method method) {
    * TODO: integrate with GapicContext for this method.
    */
   private boolean isSupported(Method method) {
-    return !method.getResponseStreaming() && !method.getRequestStreaming();
+    if (!getFeatureConfig().enableStreaming()) {
+      return !method.getResponseStreaming() && !method.getRequestStreaming();
+    }
+    return true;
```

Comment:
You could cut out one negation by starting with `if(getFeatureConfig().enableStreaming()) {return true;}`